src/scss/_mixins.scss

```diff
@@ -55,13 +55,6 @@
 		@include oTeaserHeading;
 	}
 
-	.o-teaser__heading--visited {
-		@include oColorsFor('o-teaser', text, 100);
```

Comment:
@notlee Is this part applied somewhere? We have changed heading colour `80` => `100` because there is not a big visible difference between `80` and `60`.

Reply:
Good spot @fenglish! I've updated that. Please approve the PR if it looks good to you now. :)